pkg/web/apps/apps_service.go

```diff
@@ -12,20 +12,20 @@ type (
 	}
 
 	appsService struct {
-		psqlRepository PostgreSQLRepository
+		repository Repository
 	}
 )
 
 // NewService instantiates this service
-func NewService(psqlRepository PostgreSQLRepository) Service {
+func NewService(repository Repository) Service {
 	return &appsService{
-		psqlRepository: psqlRepository,
+		repository: repository,
 	}
 }
 
 // GetApps retrieves the list of apps from the repository
 func (a *appsService) GetApps(c echo.Context) (*[]models.App, error) {
```

Comment:
same here. The server layer should really need to know about the web layer.

Reply:
Changed now